ManatiCR/fdi-front
# #2- FDI-18: Create DiscriminationFreeSpaces component.

[FDI-18](https://manati.atlassian.net/browse/FDI-18)
--
**Steps to test**
- Ensure the backend is running
- On the back-end side create 6 espacios nodes
- Go to http://localhost:8080/
- There you should be able to see them
